app/models/concerns/fails/scopes.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Fails::Scopes
+  extend ActiveSupport::Concern
+
+  module ClassMethods
+    def by_date range_as_string
+      dates = range_as_string.split(/\s*-\s*/).map do |d|
+        Timeliness.parse d rescue nil
+      end
+      start  = dates.first
+      finish = dates.last
+
+      start && finish ? where(created_at: start..finish) : all
+    end
+
+    def by_user username
+      left_outer_joins(:user)
```

Comment:
```suggestion
      left_joins(:user)
```
Es un alias, solo porque es más corto =)

Reply:
dalep. Jajajaja lo habia visto pero decia que se usa left_outer que es un alias del left_joins.